docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -138,45 +142,149 @@ You can initialize a selector with any of these objects.
+`withText(re)` | Selector | Creates selector that filters matching set by given regular expression `re`.
 
 ```js
 import { expect } from 'chai';
 import { Selector } from 'testcafe';
 
-fixture `My fixture`
-    .page `http://www.example.com/`;
 
-const removeButton = Selector(cl => document.getElementsByClassName(cl), { text: 'Remove' });
-const secondButton = Selector(cl => document.getElementsByClassName(cl), { index: 1 });
+fixture `Example page`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('My test', async t => {
+    const secondCheckBox = Selector('input[type=checkbox]').nth(1);
+    const windowsLabel   = Selector('label').withText('Windows');
 
-test('A shadowed remove button is focused', async t => {
-    expect(await removeButton('shadowed').focused).to.be.true;
+    await t
+        .click(secondCheckBox)
+        .click(windowsLabel);
 });
+```
 
-test('The second disabled button is visible', async t => {
-    expect(await secondButton('disabled').visible).to.be.true;
+If all nodes are filtered out, the selector returns `null`.
+
+### Find Elements by DOM Hierarchy
+
+Selector provide methods to find elements by DOM hierarchy in jQuery-style.
+
+Property | Description
+------ | -----
+`find(cssSelector)` | Finds all descendants of all nodes in the matching set and filters them either by CSS selector.
+`find(filterFn)` | Finds all descendants of all nodes in the matching set and filters them using `filterFn`. `filterFn` is a client function predicate that receives node.
+`parent()` | Finds all parents of all nodes in the matching set.
+`parent(index)` | Finds all parents of all nodes in the matching set and filters them by `index` (0 is closest).
+`parent(cssSelector)` | Finds all parents of all nodes in the matching set and filters them by `cssSelector`.
+`parent(filterFn)` | Finds all parents of all nodes in the matching set and filters them by `filterFn`. `filterFn` is a client function predicate that receives node.
+`child()` | Finds all child elements (not nodes) of all nodes in the matching set.
+`child(index)` | Finds all child elements (not nodes) of all nodes in the matching set and filters them by `index`.
+`child(cssSelector)` | Finds all child elements (not nodes) of all nodes in the matching set and filters them by `cssSelector`.
+`child(filterFn)` | Finds all child elements (not nodes) of all nodes in the matching set and filters them by `filterFn`. `filterFn` is a client function predicate that receives node.
+`sibling()` | Finds all sibling  elements (not nodes) of all nodes in the matching set.
+`sibling(index)` | Finds all sibling  elements (not nodes) of all nodes in the matching set and filters them by `index`.
+`sibling(cssSelector)` | Finds all sibling  of all nodes in the matching set and filters them by `cssSelector`.
+`sibling(filterFn)` | Finds all sibling  of all nodes in the matching set and filters them by `filterFn`. `filterFn` is a client function predicate that receives node.
```

Comment:
Finds all sibling elements (not nodes) of all nodes in the matching set and filters them by `filterFn`. `filterFn` is a client function predicate that receives a node.